spec/services/course/user_invitation_service_spec.rb

```diff
@@ -387,6 +388,48 @@ def invite
             end
           end
         end
+
+        context 'when users already exist in the current instance and \
+          default course timeline setting is fomo' do
+          before do
+            course.update!(default_timeline_algorithm: 'fomo')
+          end
+          it 'sets the timeline algorithm for the users to fomo' do
+            subject.send(:invite_users, temp_csv_from_attributes(existing_users, existing_roles))
+            existing_users.each do |user|
+              found_user = course.course_users.find { |course_user| course_user.user == user }
+              expect(found_user.timeline_algorithm).to eq('fomo')
+            end
+          end
+        end
```

Comment:
@ekowidianto Can I check if this set of tests correctly target users self-enrol into course?

Reply:
Seems like there is no specific test for self enrolled course user yet. But there is a feature test which I have just added. You can take a look!